language-guides/java/spring-boot-jdbc/src/main/java/movies/spring/jdbc/SampleMovieApplication.java

```diff
@@ -129,7 +132,7 @@ public void write(byte[] buf, int off, int len) {
 
     @Bean
     public DataSource dataSource() {
-        return new DriverManagerDataSource(NEO4J_URL);
+        return new DriverManagerDataSource(NEO4J_URL, NEO4J_USER, NEO4J_PASSWORD);
```

Comment:
Lorenzo would you mind fixing this in the sample application in http://github.com/neo4j-examples?